src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -165,9 +169,10 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
 
     handleFetchError = (): void => {
         this.setState({
-            error: messages.versionFetchError,
+            error: this.props.showVersionUpsell ? messages.versionFetchErrorUpsell : messages.versionFetchError,
             isLoading: false,
             isWatermarked: false,
+            onUpgradeCTAClick: this.props.onUpgradeCTAClick,
```

Comment:
We should almost always try to avoid copying props to state. Is this unused?

Reply:
it is, will update to remove from state.